File: PL022.cpp
#include <iostream>
using namespace std;

string toLower(string s)
{
    for (int i = 0; i < s.length(); i++)
    {
        if (s[i] >= 'A' && s[i] <= 'Z')
        {
            s[i] = s[i] + 32;
        }
    }

    return s;
}

int main()
{
    string s;
    cout << "Enter the string : " << endl;
    cin >> s;

    cout << "The lower case of string '" << s << "' is : " << toLower(s) << endl;

    return 0;
}

File: PL022.java
import java.util.Scanner;

public class PL022 {
    public static void main(String[] args) {
        String s;
        System.out.println("Enter the string : ");
        Scanner scanner = new Scanner(System.in);
        s = scanner.nextLine();
        scanner.close();
        System.out.println("The lower case of given string is : " + toLower(s));
    }

    public static String toLower(String s) {

        char[] chars = s.toCharArray();
        for (int i = 0; i < chars.length; i++) {
            if (chars[i] >= 'A' && chars[i] <= 'Z') {
                chars[i] = (char) (chars[i] + 32);
            }
        }

        s = String.valueOf(chars);
        return s;
    }
}
